types/highland/index.d.ts

```diff
@@ -784,6 +831,22 @@ declare namespace Highland {
 		 */
 		map<U>(f: (x: R) => U): Stream<U>;
 
+		pick<Prop extends keyof R>(props: Prop[]): Stream<Pick<R, Prop>>;
+		/**
+		 *
+		 * Retrieves copies of all the elements in the collection
+		 * that satisfy a given predicate. Note: When using ES3,
+		 * only enumerable elements are selected. Both enumerable
+		 * and non-enumerable elements are selected when using ES5.
+		 *
+		 * @id pickBy
+		 * @section Transforms
+		 * @name Stream.pickBy(f)
+		 * @param {Function} f - the predicate function
+		 * @api public
+		 */
+		pickBy(f: (key: string, value: any) => boolean): Stream<Partial<R>>
```

Comment:
Could this be the following?

```
		pickBy<K extends keyof R>(f: (key: K, value: R[K]) => boolean): Stream<Partial<R>>
```

Reply:
Sure, that makes sense. The type of `value` isn't as useful as it could be here, because narrowing the type of `key` doesn't affect the type of `value`, but it beats an `any`.